src/lib/checkbox/_checkbox-theme.scss

```diff
@@ -63,7 +80,23 @@
   }
 
   // TODO(jelbourn): remove style for temporary ripple once the real ripple is applied.
-  .md-checkbox-focused .md-ink-ripple {
-    background-color: md-color($accent, 0.26);
+  .md-checkbox-focused {
+    &.md-primary {
+      .md-ink-ripple {
+        background-color: md-color($primary, 0.26);
+      }
+    }
```

Comment:
It would also be a little more concise as
```scss
.md-checkbox-focused {
  &.primary .md-ink-ripple {
    background: md-color($primary, 0.26);
  }
  ...
}
```